### DISCUSSION TITLE
[HELP] Permanent deploy chaos key

### DISCUSSION THREAD

**Discussion by Visalyputra** [projectdiscovery/chaos-client#67] [Q&A] [answered]
How to permanent deploy the api key using export CHAOS_KEY=CHAOS_API_KEY  because everytime i logged out on my machine i will manually input the api key when i want to use chaos. can anyone help me on this one.

**Comment by ehsandeep** [comment, marked as answer]
@ShinTakata you can put this line into `export CHAOS_KEY=CHAOS_API_KEY` your default bash profile to make it permanent.

**Comment by Visalyputra** [comment]
oh ok. i will test it after that i will confirm it if its working or not. thank you for this suggestion.

**Comment by Visalyputra** [comment]
(Update) Nope It does not work. i do not know why. i need to do a source ~/.bash_profile in order to work everytime i login  I'm using a vps server.

**Reply by ehsandeep** [reply]
You can alias then, 

```
nano ~/.bash_profile
alias chaos='chaos -key XXX'
source ~/.bash_profile
```

**Comment by Visalyputra** [comment]
It worked! Thank you so much 👍
